metrics/metrics.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package metrics
```

Comment:
I don't think this is the best name for the package, it's actually more about duration than metrics (metrics is what you've seen it used for, but that's not what it is).

Reply:
Due to these two util functions are used to generate metrics.
And definitely, more util functions for metrics can be moved here to help all k8s related projects.
So I perfer the `metrics` as the pkg name. But I'm open for a better one.